package.json

```diff
@@ -12,29 +12,28 @@
   "dependencies": {
     "amqplib": "0.5.1",
     "co": "4.6.0",
-    "joi": "10.2.1",
-    "lodash": "4.14.2"
+    "joi": "13.6.0",
+    "lodash": "4.17.5"
   },
   "devDependencies": {
     "chai": "3.5.0",
-    "co-mocha": "1.1.3",
-    "co-supertest": "0.0.10",
-    "coveralls": "2.11.16",
+    "co-mocha": "1.2.2",
+    "coveralls": "3.0.2",
```

Comment:
We are sending coverage reports to codecov so it can be removed

Reply:
ok